docs/getting_started/examples.rst

```diff
@@ -17,17 +17,11 @@ The following notebooks demonstrate how Modin can be used for scalable data scie
 Tutorials
 '''''''''
 
-PandasOnRay
-"""""""""""
-
 The following tutorials cover the basic usage of Modin. `Here <https://www.youtube.com/watch?v=NglkafEmbhE>`_ is a one hour video tutorial that walks through these basic exercises.
 
-- Exercise 1: Introduction to Modin [`Source <https://github.com/modin-project/modin/blob/master/examples/tutorial/jupyter/execution/pandas_on_ray/local/exercise_1.ipynb>`_]
-- Exercise 2: Speed Improvements with Modin [`Source <https://github.com/modin-project/modin/blob/master/examples/tutorial/jupyter/execution/pandas_on_ray/local/exercise_2.ipynb>`_]
-- Exercise 3: Defaulting to pandas with Modin [`Source <https://github.com/modin-project/modin/blob/master/examples/tutorial/jupyter/execution/pandas_on_ray/local/exercise_3.ipynb>`_]
-
-PandasOnRay
-"""""""""""
+- Exercise 1: Introduction to Modin [`Source PandasOnRay <https://github.com/modin-project/modin/blob/master/examples/tutorial/jupyter/execution/pandas_on_ray/local/exercise_1.ipynb>`_, `Source OmnisciOnNative <https://github.com/modin-project/modin/blob/master/examples/tutorial/jupyter/execution/omnisci_on_native/local/exercise_1.ipynb>`_]
```

Comment:
These links should be added in [Tutorials](https://modin.readthedocs.io/en/latest/usage_guide/examples/index.html#tutorials) section of the usage guide too.

Reply:
@YarShev This was added